data/db.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I am not sure we have external dependencies for the service but this could be removed and probably serverless-json too as a dependency